apps/docs/src/app/core/component-docs/message-toast/message-toast-docs.module.ts

```diff
@@ -22,8 +22,15 @@ const routes: Routes = [
     }
 ];
 
+const messageToastConfig = new MessageToastConfig();
+messageToastConfig.duration = 5000;
```

Comment:
As for me it makes sense to be able to pass parameters to the MessageToastConfig istead of mutating it after creation.